spec/rails_helper.rb

```diff
@@ -6,6 +6,12 @@
 require "capybara/rspec"
 require "capybara/poltergeist"
 require "capybara-screenshot/rspec"
+
+# This will insert a <base> tag with the asset host into the pages created by
+# save_and_open_page, meaning that relative links will be loaded from the
+# development server if it is running.
+Capybara.asset_host = "http://localhost:4000"
```

Comment:
@mapreal19 Why is this port 4000 and not port 3000? 

We switched to using port 3000 for the rails server, right? and port 4000 is for the hot reload server? And how does it work given that assets during development are coming from port 3500?